src/components/PeoplePage.tsx

```diff
@@ -1,33 +1,92 @@
-import { PeopleFilters } from './PeopleFilters';
+import { useEffect, useState } from 'react';
 import { Loader } from './Loader';
+import { Person } from '../types';
 import { PeopleTable } from './PeopleTable';
+import { Outlet, useSearchParams } from 'react-router-dom';
 
 export const PeoplePage = () => {
-  return (
-    <>
-      <h1 className="title">People Page</h1>
+  const [data, setData] = useState<Person[] | null>(null);
+  const [error, setError] = useState<string | null>(null);
+  const [loading, setLoading] = useState(false);
+  const [searchParams] = useSearchParams();
 
-      <div className="block">
-        <div className="columns is-desktop is-flex-direction-row-reverse">
-          <div className="column is-7-tablet is-narrow-desktop">
-            <PeopleFilters />
-          </div>
+  useEffect(() => {
+    setLoading(true);
 
-          <div className="column">
-            <div className="box table-container">
-              <Loader />
+    fetch('/api/people.json')
```

Comment:
```suggestion
    fetch('./api/people.json')
```
on demo link load people is failed.
wrong  request url.
It is right url
https://andrew1256.github.io/react_people-table-advanced/api/people.json
![Screenshot 2025-05-17 at 09 31 48](https://github.com/user-attachments/assets/60aa863e-423b-4dc1-be02-7253de60c795)